bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/s3/S3BagVerifier.scala

```diff
@@ -36,6 +36,8 @@ trait S3BagVerifier[B <: BagVerifyContext[S3ObjectLocationPrefix]]
     location: S3ObjectLocation
   ): String =
     location.key.replace(root.keyPrefix, "")
+
+  override def isRetriable(error: BagVerifierError): Boolean = false
```

Comment:
Maybe a quick comment about why there are no retryable S3 errors? (Because we've got that retrying logic elsewhere, presumably?)